projects/go-sdk/handlers/searchEvents.go

```diff
@@ -48,28 +63,123 @@ func SearchEvents(w http.ResponseWriter, r *http.Request) {
 			suspect = &val
 		}
 	}
+	if query.Has("vpn") {
+		if val, err := strconv.ParseBool(query.Get("vpn")); err == nil {
+			vpn = &val
+		}
+	}
+	if query.Has("virtualMachine") {
+		if val, err := strconv.ParseBool(query.Get("virtualMachine")); err == nil {
+			virtualMachine = &val
+		}
+	}
+	if query.Has("tampering") {
+		if val, err := strconv.ParseBool(query.Get("tampering")); err == nil {
+			tampering = &val
+		}
+	}
+	if query.Has("antiDetectBrowser") {
+		if val, err := strconv.ParseBool(query.Get("antiDetectBrowser")); err == nil {
+			antiDetectBrowser = &val
+		}
+	}
+	if query.Has("incognito") {
+		if val, err := strconv.ParseBool(query.Get("incognito")); err == nil {
+			incognito = &val
+		}
+	}
+	if query.Has("privacySettings") {
+		if val, err := strconv.ParseBool(query.Get("privacySettings")); err == nil {
+			privacySettings = &val
+		}
+	}
+	if query.Has("jailbroken") {
+		if val, err := strconv.ParseBool(query.Get("jailbroken")); err == nil {
+			jailbroken = &val
+		}
+	}
+	if query.Has("frida") {
+		if val, err := strconv.ParseBool(query.Get("frida")); err == nil {
+			frida = &val
+		}
+	}
+	if query.Has("factoryReset") {
+		if val, err := strconv.ParseBool(query.Get("factoryReset")); err == nil {
+			factoryReset = &val
+		}
+	}
+	if query.Has("clonedApp") {
+		if val, err := strconv.ParseBool(query.Get("clonedApp")); err == nil {
+			clonedApp = &val
+		}
+	}
+	if query.Has("emulator") {
+		if val, err := strconv.ParseBool(query.Get("emulator")); err == nil {
+			emulator = &val
+		}
+	}
+	if query.Has("rootApps") {
+		if val, err := strconv.ParseBool(query.Get("rootApps")); err == nil {
+			rootApps = &val
+		}
+	}
+	if query.Has("minSuspectScore") {
+		if score, err := strconv.ParseFloat(query.Get("minSuspectScore"), 32); err == nil {
+			scoreFloat32 := float32(score)
+			minSuspectScore = &scoreFloat32
+		}
+	}
+    if query.Has("ipBlocklist") {
```

Comment:
nit: indentation is not the same with previous code

Reply:
Fixed